app/lib/account_deletion_config.rb

```diff
@@ -1,27 +1,38 @@
 # frozen_string_literal: true
 
-module AccountDeletionConfig
-  module_function
+class AccountDeletionConfig
+  def self.configure(config)
+    @configuration = new(config)
+  end
+
+  def self.config
+    (@configuration || new).config
```

Comment:
shouldn't be here `@configuration || configure({})` ?

no not initialize everyt time you call `AccountDeletionConfig.config` ?

Reply:
Good idea 👍